stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +145,177 @@ private static String getIdAttribute(View element) {
 
   @Override
   public View getViewForHighlighting(Object element) {
-    return (View)element;
+    return (View) element;
+  }
+
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+
+    for (ViewCSSProperty property : getProperties()) {
+      try {
+        getCSSStringFromValue(
+            element,
+            property.getCSSName(),
+            property.getValue(element),
+            property.getAnnotation(),
+            styles);
+      } catch (IllegalAccessException | InvocationTargetException e) {
+        LogUtil.e(e, "failed to get style property" + property.getCSSName() +
+            " of element id=" + getIdAttribute(element));
```

Comment:
I believe elements can end up with the same ID value, or even just `View.NO_ID` (which is zero or -1, I forget), so this won't always give enough context to track down an error. I recommend using element.toString(), which will include the ID and other stuff which Android UI devs should be accustomed to parsing. e.g.

```
LogUtil.e(e, "failed to get style property " + property.getCSSName() +
    "of element " + element.toString());
```